packages/hebao_v1/contracts/modules/security/GuardianUtils.sol

```diff
@@ -65,76 +57,40 @@ library GuardianUtils
             require(!walletOwnerSigned, "WALLET_OWNER_SIGNATURE_NOT_ALLOWED");
         }
 
-        // Update the signingGuardians array with the actual number of guardians that have signed
-        // (could be 1 less than the length if the owner signed as well)
-        assembly { mstore(signingGuardians, numGuardians) }
-        uint[MAX_NUM_GROUPS] memory signed = countGuardians(signingGuardians);
+        uint numExtendedSigners = allGuardians.length;
+        uint numGuardianApprovals = signers.length;
 
-        // Count the number of votes
-        uint totalNumVotes = 0;
-        uint numVotes = 0;
-        if (requirement != SigRequirement.OwnerNotAllowed) {
-            totalNumVotes += 1;
-            numVotes += walletOwnerSigned ? 1 : 0;
+        if (walletOwnerSigned) {
```

Comment:
@Brechtpd please review and verify the logic is correct : https://github.com/Loopring/protocols/issues/1924